components/renku_data_services/session/api.spec.yaml

```diff
@@ -290,19 +270,53 @@ components:
           $ref: "#/components/schemas/ContainerImage"
         default_url:
           $ref: "#/components/schemas/DefaultUrl"
+        owner_type:
+          $ref: "#/components/schemas/EnvironmentOwnerType"
+        owner_id:
+          $ref: "#/components/schemas/EnvironmentOwnerId"
+        uid: 
+          $ref: "#/components/schemas/EnvironmentUid"
+        gid: 
+          $ref: "#/components/schemas/EnvironmentGid"
+        working_directory:
+          $ref: "#/components/schemas/EnvironmentWorkingDirectory"
+        mount_directory:
+          $ref: "#/components/schemas/EnvironmentMountDirectory"
+        port:
+          $ref: "#/components/schemas/EnvironmentPort"
+        visibility:
+          $ref: "#/components/schemas/EnvironmentVisibility"
       required:
         - id
         - name
         - creation_date
         - container_image
+        - official
+        - port
+        - working_directory
+        - mount_directory
+        - uid
+        - gid
+        - default_url
+        - owner_type
+        - owner_id
+        - visibility
       example:
         id: 01AN4Z79ZS6XX96588FDX0H099
         name: JupyterLab environment
         creation_date: "2023-11-01T17:32:28Z"
         description: JupyterLab session environment
         container_image: renku-jupyter:latest
         default_url: "/lab"
-    EnvironmentPost:
+        port: 8080
+        working_directory: /home/jovyan/work
+        mount_directory: /home/jovyan/work
+        uid: 1000
+        gid: 1000
+        owner_type: user
```

Comment:
To add to the argument here: the corresponding fields are called `namespace` in the project entity. Wr should hold off until we agree on a consistent API for all entities before introducing this change. Note that I think that the owner fields offer better information to the ApI consumer, but then the project API should be updated too.